src/test/kotlin/ContainerUnitTest.kt

```diff
@@ -237,6 +238,60 @@ class ContainerUnitTest {
         }
     }
 
+    private val getContentFromRCCases = listOf(
+        "sng_book_various_content_rc",
+        "sng_book_various_content_rc.zip"
+    )
+
+    @Test
+    @Throws(Exception::class)
+    fun getContentFromRC() {
+        getContentFromRCCases.forEach {
+            val classLoader = this.javaClass.classLoader
+            val resource = classLoader.getResource(it)
+            val rcPath = File(resource!!.toURI().path)
+            val tempDir = createTempDir()
+            val rc = ResourceContainer.load(rcPath)
+            val wavContent = rc.getProjectContent(
+                    projectIdentifier = "sng", extension = "wav"
+            )
+
+            assertNotNull(wavContent)
+            assertEquals(2, wavContent!!.streams.size)
+
+
+            wavContent.streams.forEach { entry ->
+                tempDir.resolve(File(entry.key).name).outputStream().use { output ->
+                   output.write(entry.value.read())
+                }
+            }
+
+            val usfmContent = rc.getProjectContent(extension = "usfm")
+
+            assertNotNull(usfmContent)
+            assertEquals(1, usfmContent!!.streams.size)
```

Comment:
Okay so this is testing that you have one USFM file.

Sounds like we should make some kind of test case class that has your input for the test;
file, format, number of expected streams
And use that for one generic test that includes handling this (usfm) and the previous (wav)
Look to some examples from the tests you wrote for Fetcher